tests/unit/script/htmlimageelement.rs

```diff
@@ -0,0 +1,16 @@
+
+use script::test::sizes::parse_a_sizes_attribute;
+use script::test::DOMString;
+
+#[test]
+fn empty_vector() {
+    assert!(parse_a_sizes_attribute(DOMString::new(), None).is_ok());
```

Comment:
We will want to assert that the result is an empty vector (as the test name implies ;))